build/Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+FROM golang:1.17.3-alpine3.14 AS builder
```

Comment:
if there are no strong objections let's use images without patch versions if available to get the latest patch version on the build automatically

Reply:
I haven't focused on dockerfile at all yet. I will work on that tomorrow.

Reply:
Dockerfile improvements pushed